LSPS5/README.md

```diff
@@ -0,0 +1,375 @@
+# LSPS5 Webhook Registration
+
+| Name    | `webhook_registration` |
+| ------- | ---------------------- |
+| Version | 1                      |
+| Status  | For Implementation     |
+
+## Motivation
+
+In typical mobile environments, a program that is not currently
+being focused on by the user will be suspended, with all its
+TCP connections dropped.
+
+Typically, such a program would not be able to get either CPU
+time or any ability to receive information from network.
+
+If an LSPS client is implemented as a program on such a mobile
+environment, then when it is suspended, the LSP cannot contact
+the client to perform any operations, such as to forward a
+payment to the client.
+
+Generally, mobile environments will have the mobile OS developers
+run a server or network of servers where an application developer
+can deliver "push notifications".
+When an application developer server contacts the mobile OS
+developer push notification server, the application developer
+server signs and sends a message that is to be displayed to the
+user of the mobile unit, which the mobile OS server will validate
+before causing the message to be displayed to the specified user.
+If the user then taps on this "push notification" on their screen,
+the application is awoken and it can then re-establish its TCP
+connections.
+
+There may be multiple implementations of mobile LSPS-supporting
+clients, each of which run their own application developer server
+and have their own signing keys for these push notifications.
+
+This specification provides a way for a mobile client to
+register some specific webhook by which the LSP can signal a
+push notification to the application developer server, which
+will in turn convert the push notification to one that it itself
+signs and can send to the mobile OS developer server.
+
+### Actors
+
+The 'client' is a specific instance of the client application
+running on some mobile device.
+
+The 'LSP' is a published node that may need to awaken some
+client instance in order to handle some event.
+
+The 'notification delivery service' is the server that a mobile
```

Comment:
I am not particularly seeing the value of this acronym, but then I suppose that even as a completely human non-AI with actual meat fingers I am more willing to type in entire definitions like notification delivery service rather than acronyms.

Reply:
Ack NDS